### PR TITLE
added resx to create/edip position

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

File: test/PositionService.Business.UnitTests/Position/EditPositionCommandTests.cs [hunk 1]
```diff
@@ -122,7 +121,7 @@ public async Task PositionIsNullTest()
     {
       OperationResultResponse<bool> result = new(
         body: false,
-        errors: new List<string>() {});
+        errors: new List<string>() { });
```
Comment (SolovyovDmitry) [inline test/PositionService.Business.UnitTests/Position/EditPositionCommandTests.cs:124]: Are these brackets required at all?

**Review by SolovyovDmitry — APPROVED** [review]

File: src/PositionService.Validation/Position/Resources/PositionRequestValidationResource.resx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    <value>Description is to long</value>
+  </data>
+  <data name="IncorrectType" xml:space="preserve">
+    <value>Incorrect format of </value>
+  </data>
+  <data name="NameExists" xml:space="preserve">
+    <value>The position name already exists</value>
+  </data>
+  <data name="NameLong" xml:space="preserve">
+    <value>Name is to long</value>
```
Comment (ndoriomedov) [inline src/PositionService.Validation/Position/Resources/PositionRequestValidationResource.resx:130]: Is too

File: src/PositionService.Validation/Position/Resources/PositionRequestValidationResource.resx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="DescriptionLong" xml:space="preserve">
+    <value>Description is to long</value>
```
Comment (ndoriomedov) [inline src/PositionService.Validation/Position/Resources/PositionRequestValidationResource.resx:121]: Too

**Review by ndoriomedov — APPROVED** [review]

File: src/PositionService.Validation/Position/Resources/PositionRequestValidationResource.resx (hunk 2 above)
Comment (VLiamin) [inline src/PositionService.Validation/Position/Resources/PositionRequestValidationResource.resx:130]: too

File: src/PositionService.Validation/Position/Resources/PositionRequestValidationResource.resx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+  </data>
+  <data name="IncorrectType" xml:space="preserve">
+    <value>Incorrect format of </value>
```
Comment (VLiamin) [inline src/PositionService.Validation/Position/Resources/PositionRequestValidationResource.resx:124]: of?

**Review by VLiamin — APPROVED** [review]

File: src/PositionService.Validation/Position/Resources/PositionRequestValidationResource.resx (hunk 4 above)
Comment (MariyaSin) [reply src/PositionService.Validation/Position/Resources/PositionRequestValidationResource.resx:124]: property name adds in validator

**Pull request merged**